demisto_sdk/commands/content_graph/objects/content_item.py

```diff
@@ -75,13 +76,60 @@ def match(_dict: dict, path: Path) -> bool:
     def pack_id(self) -> str:
         return self.in_pack.pack_id if self.in_pack else ""
 
+    @validator("pack", always=True)
+    def validate_pack(cls, v: Any, values) -> Optional["Pack"]:
+        # Validate that we have the pack containing the content item.
+        # The pack is either provided directly or needs to be located.
+
+        pack = cls.get_pack(values.get("relationships_data"), values.get("path"))
+        if v and not isinstance(v, fields.FieldInfo):
+            return v
+        return pack
+
     @property
-    def support_level(self) -> str:
-        return (
-            self.in_pack.support_level
-            if self.in_pack and self.in_pack.support_level
-            else ""
-        )
+    def in_pack(self) -> Optional["Pack"]:
+        """
+        This returns the Pack which the content item is in.
+
+        Returns:
+            Pack: Pack model.
+        """
+        if not self.pack:
+            self.pack = ContentItem.get_pack(self.relationships_data, self.path)
+        return self.pack  # type: ignore[return-value]
+
+    @staticmethod
+    def get_pack(
+        relationships_data: dict,
+        path: Path,
+    ) -> Optional["Pack"]:
+        """
+        Returns the Pack which the content item is in.
+
+        Returns:
+            Pack: Pack model.
+        """
+        pack = None
+        if in_pack := relationships_data[RelationshipType.IN_PACK]:
+            pack = next(iter(in_pack)).content_item_to  # type: ignore[return-value]
+        if not pack:
+            if pack_name := get_pack_name(path):
+                pack = BaseContent.from_path(
+                    CONTENT_PATH / PACKS_FOLDER / pack_name, metadata_only=True
+                )  # type: ignore[assignment]
+        return pack  # type: ignore[return-value]
+
+    @validator("support", always=True)
```

Comment:
let's move up below the other validators, just for better organization of the code